src/css/main.css

```diff
@@ -53,6 +53,41 @@ a.disabled {
     color: #999;
 }
 
+/* The slider (use -webkit- (Chrome, Opera, Safari, Edge) and -moz- (Firefox) and -ms- (IExplorer) to override default look) */
+input[type=range] {
+  -webkit-appearance: none;
+  /*height: 30px;*/
+  /*margin: 10px 0;*/
+  width: 100%;
+}
+input[type=range]:focus {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Outlines shouldn't be hidden unless other visual changes are made.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5576260)